Mathlib/NumberTheory/Padics/PadicNumbers.lean

```diff
@@ -609,7 +608,6 @@ theorem defn (f : PadicSeq p) {ε : ℚ} (hε : 0 < ε) :
     rw [PadicSeq.norm, dif_pos h] at hge
     exact not_lt_of_ge hge hε
   unfold PadicSeq.norm at hge; split_ifs at hge
-  · exact not_le_of_gt hε hge
```

Comment:
The proof now matches the mathlib3 version more closely: https://github.com/leanprover-community/mathlib/blob/65a1391a0106c9204fe45bc73a039f056558cb83/src/number_theory/padics/padic_numbers.lean#L505